src/Common/src/Microsoft/Win32/SafeHandles/SafeX509NameHandle.Unix.cs

```diff
@@ -27,4 +27,25 @@ public override bool IsInvalid
             get { return handle == IntPtr.Zero; }
         }
     }
+
+    [SecurityCritical]
+    internal sealed class SafeX509NameStackHandle : SafeHandle
```

Comment:
Common\src\Interop\Unix\System.Security.Cryptography.Native\Interop.X509Name.cs seems like a better location for this.